docs/_docs/liquid/tags.md

```diff
@@ -19,6 +19,13 @@ thanks to [Rouge](http://rouge.jneen.net). Rouge is the default highlighter
 in Jekyll 3 and above. To use it in Jekyll 2, set `highlighter` to `rouge`
 and ensure the `rouge` gem is installed properly.
 
+<div class="note warning">
+  <h5>Jekyll 4 Pygments support</h5>
+  <p>
+    Pygments is not supported in Jekyll 4.
```

Comment:
The support has not been removed yet. The PR for that is yet to be merged.
I recommend refraining from mentioning anything about Jekyll - 4.0 in the current docs (meant for v3.x)

Reply:
Yeah agreed, when Jekyll 4.0 is imminent someone will need to go through the docs adding this sort of warning.

Reply:
We'll have it listed visibly inside `Upgrading to v4.0` document..